Project.toml

```diff
@@ -19,18 +19,18 @@ Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```

Comment:
Neither revise nor StableRNGs need to be in project. If StableRNGs needs to be in Project.toml, it must be a test-only dependency.